### PR TITLE
Updating 4.13+ and main branches per new architecutre guidelines

### PR DESCRIPTION
**Pull request by kelbrown20**
**For versions** 4.13+ 

**Description:** Updating main and then cherrypicking to 4.13. New architecture guidelines in [PR#295](https://github.com/redhat-documentation/supplementary-style-guide/pull/295)

**Previews:** 
- [Configuring multi-architecture complete machines on an OpenShift Container Platform cluster ](https://59279--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/multi-architecture-configuration.html)
  1. [Creating a cluster with multi-architecture compute machine on Azure](https://59279--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/multi-architecture-configuration.html#creating-a-cluster-with-multi-architecture-compute-machine-on-azure) 
  2. [Creating an ARM boot image using the Azure image gallery](https://59279--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/multi-architecture-configuration.html#multi-architecture-creating-arm64-bootimage_multi-architecture-configuration) 
  3. [Adding a multi-architecture compute machine set to your cluster ](https://59279--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/multi-architecture-configuration.html#multi-architecture-modify-machine-set_multi-architecture-configuration)
- [Selecting an installation method and preparing a cluster -> Support installation methods for different platforms ](https://59279--docspreview.netlify.app/openshift-enterprise/latest/installing/installing-preparing.html#installing-preparing-selecting-cluster-type)
- Installing a user provisioned cluster on bare metal 
  1. [Installing a RHCOS by using RXE or IPXE booting](https://59279--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-bare-metal.html#installation-user-infra-machines-pxe_installing-bare-metal)
  4. [Advanced RHCOS installation configuration -> Advanced RHCOS installation reference](https://59279--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal/installing-bare-metal.html#installation-user-infra-machines-advanced_installing-bare-metal)
- Installing on-premise with Assisted installer 
  1. [Installing an on premise cluster using the Assisted installer ](https://59279--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_on_prem_assisted/installing-on-prem-assisted.html#using-the-assisted-installer_installing-on-prem-assisted)
  5. Installing with the assisted installer -> Setting the cluster details [also missing]
- Installing a cluster on AWS in a restricted network with user provisioned infrastructure
  1. [Accessing the RHCOS AMI with stream meta data ](https://57214--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-restricted-networks-aws.html#installation-aws-ami-stream-metadata_installing-restricted-networks-aws) Also need to fix this one 
  2. [RHCOS AMI for the AWS infrastructure](https://57214--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-restricted-networks-aws.html#installation-aws-user-infra-rhcos-ami_installing-restricted-networks-aws) 
- Installing a cluster on AWS with customizations 
  1. [Tested instance type AWS on 64-bit ARM](https://57214--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations.html#installation-aws-tested-machine-types_installing-aws-customizations)
  2. [Tested instance type AWS](https://57214--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-customizations.html#installation-aws-arm-tested-machine-types_installing-aws-customizations)
- Installing a cluster on Azure with customization 
  1. [Tested instance type for Azure on 64-bit ARM infrastructures](https://59279--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-customizations.html#installation-azure-arm-tested-machine-types_installing-azure-customizations) 
  2. [Tested instance types for Azure](https://59279--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-customizations.html#installation-azure-tested-machine-types_installing-azure-customizations) 
  6. [Selecting an Azure Marketplace image ](https://59279--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-customizations.html#installation-azure-marketplace-subscribe_installing-azure-customizations)
- [Troubleshooting operating system issues](https://59279--docspreview.netlify.app/openshift-enterprise/latest/support/troubleshooting/troubleshooting-operating-system-issues.html#investigating-kernel-crashes) 
- Installing a cluster on AWS using CloudFormation 
  1. [Creating the control plane machines in AWS](https://59279--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-user-infra.html#installation-creating-aws-control-plane_installing-aws-user-infra) 
  2. [Creating the worker nodes in AWS](https://59279--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-user-infra.html#installation-creating-aws-worker_installing-aws-user-infra) 
- Pipeline CLI (tkn) -> Installing tkn 
  1. [Installing the Red Hat OpenShift Pipelines CLI on Linux](https://59279--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/tkn_cli/installing-tkn.html#installing-tkn-on-linux)  
  2. [Installing the Red Hat OpenShift Pipelines CLIU on Linux using the RPM](https://59279--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/tkn_cli/installing-tkn.html#installing-tkn-on-linux-using-rpm) 

**QE review:** 
- [x] QE approval

### CODE REVIEW

File: modules/installation-aws-arm-tested-machine-types.adoc [hunk 1]
```diff
@@ -10,16 +10,16 @@
 // installing/installing_aws/installing-restricted-networks-aws.adoc
 
 [id="installation-aws-arm-tested-machine-types_{context}"]
-= Tested instance types for AWS ARM
+= Tested instance types for AWS on 64-bit ARM infrastructures
 
-The following Amazon Web Services (AWS) ARM instance types have been tested with {product-title}.
+The following Amazon Web Services (AWS) ARM64 instance types have been tested with {product-title}.
```
Comment (mjpytlak) [inline modules/installation-aws-arm-tested-machine-types.adoc:15]: Is `ARM64` how AWS refers to these instance types. This is probably me not just understanding the guidelines you are applying, but I am seeing `64-bit ARM instances` in `installing/installing_preparing.adoc`

**Review by mjpytlak — COMMENTED** [review]
@kelbrown20 seem straight forward enough, but I have a question. I probably just do not understand the new architecture guidelines, but I noticed that `64-bit ARM`, `AArch64`, `aarch64`, and `ARM64` all seem to be used interchangeably. Can you point me to the guidelines around the correct usage?

Update: Sorry. I see that you link to the guidelines. Looking at those now.

**Comment by kalexand-rh** [comment]
The `branch/enterprise-4.14` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.13. And any PR going into `main` must also target the latest version branch (enterprise-4.14).

If the update in your PR does NOT apply to version 4.14 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

File: modules/installation-aws-arm-tested-machine-types.adoc (hunk 1 above)
Comment (kelbrown20) [reply modules/installation-aws-arm-tested-machine-types.adoc:15]: AWS uses a few formats, I've seen 64-bit Arm, ARM64, and arm64. But I think I'll change it to 64-bit ARM to be more general.

File: modules/assisted-installer-setting-the-cluster-details.adoc [hunk 2]
```diff
@@ -35,7 +35,7 @@ The base domain must be a valid DNS name. You must not have a wild card domain s
 
 . Optional: The {ai-full} already has the pull secret associated to your account. If you want to use a different pull secret, select *Edit pull secret*.
 
-. Optional: {ai-full} defaults to using `x86_64` CPU architecture. If you are installing {product-title} on `arm64` CPUs, select *Use arm64 CPU architecture*. Keep in mind, some features are not available with `arm64` CPU architecture.
+. Optional: {ai-full} defaults to using x86_64 CPU architecture. If you are installing {product-title} on 64-bit ARM CPUs, select *Use arm64 CPU architecture*. Keep in mind, some features are not available with ARM64 CPU architecture.
```
[the remark this answers is not in the document]
Comment (kelbrown20) [reply modules/assisted-installer-setting-the-cluster-details.adoc:38]: 64-bit ARM and AArch64 are kinda interchangeable, some sentences need a more specific use case. But you're right that the ARM64 format should not be used here. I'll update

**Pull request merged**